src/form.js

```diff
@@ -154,18 +154,18 @@ define([
     Form.prototype = {
         isValidHashtag: function(tag) {
             tag = this.normalizeHashtag(tag);
-            return this.hashtagDictionary.hasOwnProperty(tag) && _.isString(this.hashtagDictionary[tag]);
+            return this.hashtagDictionary.hasOwnProperty(tag) && !_.isFunction(this.hashtagDictionary[tag]);
```

Comment:
https://github.com/dimagi/Vellum/pull/678/commits/2ccf9b62153c5e0c1183584f32335b317be2a0d7

I don't entirely love this change, as it adds a good deal of repetitive code, but agree that the dictionary with mixed strings & functions is ugly.